src/pages/Map.tsx

```
@@ -1,5 +1,28 @@
+import { useState } from 'react';
+
+import { SearchInput } from '@/components/common/SearchInput';
 import { NaverMap } from '@/components/features/NaverMap';
 
 export const Map = () => {
```

Biome

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)